web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerWfs.java

```diff
@@ -0,0 +1,35 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+public class LinkCheckerWfs implements LinkCheckerInterface {
+    private static String WFS_GET_FEATURES = "?service=WFS&version=1.0.0&request=GetFeature&maxFeatures=1&outputFormat=csv&typeName=";
+
+    private String getCheckUrl(Element onlineResource) {
+        String url = LinkCheckerUtils.parseOnlineResource(onlineResource, LinkCheckerUtils.URL_XPATH);
```

Comment:
This is *very* similar to `LinkCheckerWms.getCheckUrl()` - can probably be refactored.